packages/quantic/force-app/examples/main/lwc/actionPerformRecommendationSearch/actionPerformRecommendationSearch.html

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="slds-grid slds-gutters">
+    <lightning-button
+      label="Perform recommendation search"
+      title="Perform recommendation search"
+      onclick={handleRecommendationPerformSearch}
+      disabled={disabled}
+      class="configurator__perform-recommendation-search">
```

Comment:
please opt for simple names and avoid mixing search with recommendations as they are two different use cases.
```suggestion
      class="configurator__get-recommendations">
```